cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -162,6 +162,20 @@
             "size": "medium"
           }
         },
+        {
+          "widget-type": "select",
```

Comment:
For true/false, better to use `toggle` type. See https://github.com/cdapio/hydrator-plugins/blob/develop/core-plugins/widgets/File-batchsource.json#L52 for example.

Reply:
Done.